src/code/action-types.js

```diff
@@ -0,0 +1,33 @@
+const actionTypes = {
+  SESSION_STARTED: "Session started",
+  AUTHORING_CHANGED: "Authoring changed",
+  LOADED_CHALLENGE_FROM_AUTHORING: "Loaded challenge from authoring",
+  NAVIGATED: "Navigated",
+  CHALLENGE_COMPLETE: "Challenge completed",
+  BRED: "Bred",
+  ALLELE_CHANGED: "Allele changed",
+  SEX_CHANGED: "Sex changed",
+  GAMETE_CHROMOSOME_ADDED: "Gamete chromosome added",
```

Comment:
That action existed before the parentGametes module and is mediated by the separate gametes reducer. I thought about combining the gametes reducer/actions into the parentGametes module but doing so didn't seem like it was necessarily the highest priority.